src/main/java/com/streamr/client/StreamrRESTClient.java

```diff
@@ -177,6 +178,10 @@ public UserInfo getUserInfo() throws IOException {
         return get(url, publishersJsonAdapter).getAddresses();
     }
 
+    public boolean isPublisher(String streamId, Address address) throws IOException {
+        return isPublisher(streamId, address.toString());
```

Comment:
Looks like this `toString` isn't necessary: https://github.com/streamr-dev/streamr-client-java/blob/StreamLayer-V32/src/main/java/com/streamr/client/StreamrRESTClient.java#L181-L183

Reply:
Hmm, your link points to the same code section where you commented?

Any any case, as this is a public method, I thought it's worth offering both overloaded variants to avoid breaking any applications that use this.
`isPublisher(String, Address)`
`isPublisher(String, String)`

Here, the former method calls the latter by converting the given `Address` to a `String`.